packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -92,6 +95,25 @@ const errorClass = useErrorClass(showError);
             :maxlength="props.maxlength"
             :aria-label="props.hideLabel ? props.label : undefined"
             :title="props.hideLabel ? props.label : undefined"
+            @focus="isFocused = true"
+            @blur="isFocused = false"
```

Comment:
Would prefer a CSS only solution, e.g.:

```css
.onyx-input {
  // styles for not focused input...

  &:has(&_native:focus) {
    // styles for focussed input...
}
}
```

Reply:
I use isFocused for the Icons. It's not possible with only css or?

Reply:
Sure :) I created #2252 on top of your PR with the changes